README.md

```diff
@@ -103,7 +103,7 @@ The following options are currently available.
 | --- | --- | --- | --- |
 | `enable_snippets` | `bool` | `false` | Enables snippet completions when the client also supports them. |
 | `enable_unused_variable_warnings` | `bool` | `false`| Enables warnings for local variables that aren't used. |
-| `enable_import_embedfile_argument_completions` | `bool` | `false` | Whether to enable import/embedFile argument completions |
```

Comment:
You swapped out the wrong option in the README! :P

Reply:
sorry ( ͡° ʖ̯ ͡°)